pouchDB/pouch.d.ts

```diff
@@ -1,230 +1,134 @@
-// Type definitions for Pouch 0.1
+// Type definitions for PouchDB 5.0.0
 // Project: http://pouchdb.com
-// Definitions by: Bill Sears <https://github.com/MrBigDog2U/>
```

Comment:
I write the definition complete new. Should we store the old one in a seperately file?